The path examples are incorrect and misleading

surajmb opened the pull request:
The paths mentioned http://contoso.com/video* and http://contoso.com/images* are misleading as there is a "/" missing in front of the *.
This contradicts the following statement.

PathPattern: This setting is a list of path patterns to match. Each must start with / and the only place a "*" is allowed is at the end following a "/." The string fed to the path matcher does not include any text after the first? or #, and those chars are not allowed here.

azurecla commented:
Hi __@ChrisCompy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

surajmb committed:
The path examples are incorrect and misleading

The paths mentioned http://contoso.com/video* and http://contoso.com/images* are misleading as there is a "/" missing in front of the *.
This contradicts the following statement.

PathPattern: This setting is a list of path patterns to match. Each must start with / and the only place a "*" is allowed is at the end following a "/." The string fed to the path matcher does not include any text after the first? or #, and those chars are not allowed here.
